docs/content/patterns/search-filters/index.mdx

```diff
@@ -3,7 +3,7 @@ title: Search filters
 description: Search filters help users find what they’re looking for by displaying options that meet specified criteria.
 figmaTemplateNodeId: 18586-56265
 githubTemplatePath: /.storybook/stories/DataFiltering
-storybookPath: story/patterns-search-filters--cards
+storybookPath: /story/patterns-search-filters--cards
```

Comment:
Why isn't this aligned? 🤷‍♂️ https://design-system.agriculture.gov.au/storybook/index.html?path=/story/patterns-search-filters--cards 
<img width="935" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/ea08c888-2439-4905-82a3-7d2671a05737">

Reply:
Oh, prod hasn't updated yet…?